### PR TITLE
Add doc on setting adapter options in Repo configuration.

### PR DESCRIPTION
**Pull request by starbelly**
I had to set the unamed prepared statements option for the postgres adapter earlier today and it was a bit confusing on how to do just that via config. So, here's some doc ❤️ 

  Notes: 

  - Not sure this is the best place for it, it felt right :)
  - Not sure if there needs to be more elaboration

### CODE REVIEW

File: guides/Getting Started.md [hunk 1]
```diff
@@ -72,6 +72,18 @@ config :friends, Friends.Repo,
...
+It should be noted that some adapter options may be set here. For example, suppose you have a setup that requires unamed prepared statements  The Postgres adapter has an option for this and can be set in your repo config.
+
+```elixir
+config :friends, Friends.Repo,
+  adapter: Ecto.Adapters.Postgres,
+  database: "friends_repo",
+  username: "user",
+  password: "pass",
+  hostname: "localhost"
+  prepare: :unamed
```
Comment (fishcakez) [inline guides/Getting Started.md]: I'm not sure if this is the best spot, likely it should go in `Ecto.Adapters.Postgres`. Also the correct spelling is `:unnamed`.

**Comment by josevalim** [comment]
Awesome @starbelly! Can you please add it to `Ecto.Adapters.Postgres` as @fishcakez suggested? Thanks.

**Comment by starbelly** [comment]
N/P... will do that and rebase.

**Comment by starbelly** [comment]
@josevalim @fishcakez done... Unless we want to elaborate more on adapter options.

**Comment by josevalim** [comment]
Tks @starbelly! All options here are adapter options, so I merged the lists together, thanks! 
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by starbelly** [comment]
@josevalim Awesome, more to come ❤️  Have a great weekend!

**Pull request merged**